Tribler/Core/Utilities/torrent_utils.py

```diff
@@ -2,22 +2,27 @@
 import libtorrent
 
 
-def create_torrent_file(file_path_list, params, callback=None):
+def create_torrent_file(file_path_list, params):
     fs = libtorrent.file_storage()
 
     # filter all non-files
-    file_path_list = [file for file in file_path_list if os.path.isfile(file)]
+    file_path_list_filtered = []
+    for path in file_path_list:
+        if not os.path.exists(path):
+            raise IOError('Path does not exist: %s' % path)
+        elif os.path.isfile(path):
```

Comment:
iirc, the files are also not added recursively in the current implementation in the wx GUI.